src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Reload/ReloadableProject.cs

```diff
@@ -127,9 +131,11 @@ public Task Initialize()
                         var tw = new StringWriter();
                         msbuildProject.Save(tw);
                     }
-                    catch (Microsoft.Build.Exceptions.InvalidProjectFileException)
+                    catch (Microsoft.Build.Exceptions.InvalidProjectFileException ex)
                     {
                         // Indicate we weren't able to complete the action. We want to do a normal reload
+                        var faultEvent = new FaultEvent("vs.projectsystem.projectreload.reload-failed", "Project Reload Failure due to invalid project file", ex);
```

Comment:
Ok. @srivatsn, I assumed that we actually want to record the exception as well. Is that correct? If not I'll change it to a basic event.